src/rvs_blas.cpp

```diff
@@ -26,11 +26,48 @@
 
 #include <time.h>
 #include <iostream>
+#include <cmath>
+#include <random>
+#include <thread>
+
+/* ============================================================================================ */
+// Random number generator
+using rvsblas_rng_t = std::mt19937;
+
+// Random number generator
+rvsblas_rng_t rvsblas_seed(69069); // A fixed seed to start at
```

Comment:
do we need not randomise this initial seed?

Reply:
This is just the main thread seed at init, other seeds are based on thread ids.